packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -81,6 +88,10 @@ export default createRule<Options, MessageIds>({
             type: 'boolean',
             description: `If 'true', the 'default' clause is used to determine whether the switch statement is exhaustive for union type`,
           },
+          defalutCaseCommentPattern: {
+            type: 'string',
+            description: `Regular expression to evaluate comments to mean default case`,
```

Comment:
[Docs] WDYT about this phrasing, to make it a bit more clear what the comment does?

```suggestion
            description: `Regular expression for a comment that can indicate an intentionally omitted default case.`,
```